skynet/modules/stt/streaming_whisper/state.py

```diff
@@ -28,20 +28,19 @@ def __init__(
         # before starting to drop them
         add_max_silent_chunks: int = 1,
         final_after_x_silent_chunks: int = 2,
-        force_final_duration_threshold: int = 29,
+        force_final_duration_threshold: int = 15,
         perform_final_after_silent_seconds: float = 0.8,
     ):
-        self.transcription_id = str(uuid6.uuid7())
```

Comment:
These are equivalent, except that the previous version is better because it's v7 instead of v1.